# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Result - Success result & final arm position
bool success
float64[] arm_position          # [Right_tilt, Right_lift, Right_rotate, Right_elbow]
